### PR TITLE
Update dependencies and move to ruby 3.x's docker image

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

**Review by dduportal — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 pipeline {
     agent {
         docker {
-            image 'ruby:2.4-alpine'
+            image 'ruby:3.1.3-alpine'
```
Comment (dduportal) [inline Jenkinsfile:6]:
```suggestion
            image 'ruby:3.1.3'
```

The size difference between both images is not worth the problems caused by missing tools such as `bash` (for this case where we do not use a custom image)

Comment (NotMyFault) [reply Jenkinsfile:6]: Good point!

**Comment by dduportal** [comment]
As the build #2 says `‘Jenkinsfile’ has been modified in an untrusted revision`, I've added @NotMyFault  as maintainer of the repo and triggered a repository full scan (to let Jenkins know of the permission change) + kicked a new build (`#3`) that should take the latest change in account

**Comment by NotMyFault** [comment]
Thanks! The build is green now 👏🏻

**Pull request merged**